lighthouse-core/gather/computed/trace-of-tab.js

```diff
@@ -42,21 +42,42 @@ class TraceOfTab extends ComputedArtifact {
     // *must* be stable to keep events correctly nested.
     /** @type Array<LH.TraceEvent> */
     const keyEvents = trace.traceEvents
-      .filter(e => {
-        return e.cat.includes('blink.user_timing') ||
+      .filter(e =>
+          e.cat.includes('blink.user_timing') ||
```

Comment:
generally for multi-line arrow function we keep the curly braces/`return` for clarity. Any reason to drop it here?

Reply:
done